lazy-lock.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
@Uzaaft can we add this file to gitignore?

Reply:
We can. Not sure if we should(no idea regarding the pros/cons of adding it to gitignore)

Reply:
hmm, I'm always on the latest version of plugins ( to test them out)
so this file would be changed regularly, 

not sure tbh

Reply:
I’ll add it to the gitignore.

Reply:
Fixed in 97f2d8d.